src/bokeh/core/property/either.py

```diff
@@ -81,20 +81,24 @@ def __init__(self, type_param0: TypeOrInst[Property[Any]], *type_params: TypeOrI
         super().__init__(type_param0, *type_params, default=default, help=help)
         for tp in self.type_params:
             self.alternatives.extend(tp.alternatives)
+        self.valids = []
```

Comment:
If we make the change below, then 
```suggestion
        self.valid = None
```